src/App.js

```diff
@@ -1,17 +1,26 @@
 import { HashRouter as Router, Routes, Route } from 'react-router-dom';
+import { React, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import Header from './components/header';
 import Rockets from './pages/rockets';
 import Missions from './pages/missions';
 import Profile from './pages/profile';
+import { fetchRockets } from './Rockets/rockets';
 import './App.css';
 
 function App() {
+  const rockets = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, []);
```

Comment:
I think it would be better to put the rockets logic in the rockets page.

Reply:
Thank you for your suggestion. I just made the required changes